### DISCUSSION TITLE
How to set user quota on juicefs?

### DISCUSSION THREAD

**Discussion by suzhouclark** [juicedata/juicefs#699] [Q&A] [answered]
I will use juicefs volumn as nfs share.
How to set user quotas on juicefs?

**Comment by xiaogaozi** [comment, marked as answer]
JuiceFS doesn't support per user quota, you need implement it by yourself.

**Reply by suzhouclark** [reply]
I hope your team can add this feature soon.
As an mature multi-user fs, qutoa is one of important functions.
